index.html

```diff
@@ -234,10 +234,18 @@ <h3 class="account__skills">Skills</h3>
 
     <footer class="footer">
       <nav class="footer__nav">
-        <button class="footer__button active" id="footer__home">Home</button>
-        <button class="footer__button" id="footer__bookmarks">Bookmark</button>
-        <button class="footer__button" id="footer__create">Create</button>
-        <button class="footer__button" id="footer__profile">Profile</button>
+        <button class="footer__button active" id="footer__home">
+          <img src="/img/icons/home.svg" />
+        </button>
+        <button class="footer__button" id="footer__bookmarks">
+          <img src="/img/icons/collections_bookmark.svg" />
+        </button>
+        <button class="footer__button" id="footer__create">
+          <img src="/img/icons/card_add.svg" />
+        </button>
+        <button class="footer__button" id="footer__profile">
+          <img src="/img/icons/account.svg" />
+        </button>
```

Comment:
Please don't forget the `alt` attribute in the image! Should also be descriptive like `alt="Home Page"` ...